Evaluation helper

rosalierazonable opened the pull request:
# Student Subjects Helper

- fetch student's subjects 
- returns subjects given the user's id from the session

crispaulrubi commented:
Sa MySQL query, feel nako nice if left join faculty table lang sad para makuha pd ang full name sa faculty nga gahandle sa subject. Suggestion ra ni tho

rosalierazonable commented:
> Sa MySQL query, feel nako nice if left join faculty table lang sad para makuha pd ang full name sa faculty nga gahandle sa subject. Suggestion ra ni tho

I implemented it na

> Mas nice sad guro ibalhin lang nag query sa model. para matapok tanan database queries sa model(?)

Mag involve siya ug tulo ka models idk if mas efficient na or what but I added an implementation that fetches the subjects via model nalang pud, choose whichever is efficient lang
